### PR TITLE
MariaDB to PostgreSQL migration

### PR DESCRIPTION
**Pull request by davenquinn**
This pull request seeks to convert all API routes that reference MariaDB to PostgreSQL.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,12 +11,14 @@ npm-debug.log
 node_modules
 v1/node_modules/*
 v1/npm-debug.log
-
+.github
```
Comment (davenquinn) [inline .gitignore:14]: The GitHub directory contains CI definitions and should not be ignored (at least not fully).

File: .gitignore [hunk 2]
```diff
@@ -31,5 +33,5 @@ Icon
 # Files that might appear on external disk
 .Spotlight-V100
 .Trashes
-
+.yarn
```
Comment (davenquinn) [inline .gitignore:36]: The Yarn directory also should not be fully ignored.

File: .gitignore [hunk 3]
```diff
@@ -11,12 +11,14 @@ npm-debug.log
...
-
+.github
 v1/utilities/scripts/credentials.py
+.idea
+venv
 
 credentials.ts
 !v1/credentials.ts
-!v2/credentials.ts
+v2/credentials.ts
```
Comment (davenquinn) [inline .gitignore:21]: The reason the Gitignore was structured this way for credentials is that the `v2/credentials.ts` was essentially a re-export of the "main" credentials file. Now that there's only v2, it's less important. But we should definitely ensure that two credentials files are not needed. And ideally, all credentials would be specified as environment variables or in an `.env` file

**Review by davenquinn — COMMENTED** [review]
Looks broadly good but it's hard to tell how you are specifying credentials etc. with all the files ignored. It'd also be nice to see a `Makefile` or similar specifying your local Docker command.

File: .gitignore (hunk 3 above)
Comment (amyfromandi) [reply .gitignore:20]: Right now the code references v2/credentials.ts. I commented out the rest of the other cred URI's. Will create .env once I can get a query to successfully execute from macrostrat_two in PG

**Pull request closed without merging**